pom.xml

```diff
@@ -231,6 +232,19 @@
 				</executions>
 				<configuration>
 					<defaultP2Metadata>false</defaultP2Metadata>
+					<baselineRepositories>
+						<repository>
+							<url>https://download.eclipse.org/technology/m2e/snapshots/latest/</url>
+						</repository>
+					</baselineRepositories>
+					<baselineMode>warn</baselineMode>
+					<ignoredPatterns>
+						<pattern>META-INF/ECLIPSE_.RSA</pattern>
+						<pattern>META-INF/ECLIPSE_.SF</pattern>
+						<pattern>**/org/fusesource/jansi/internal/native/Mac/*/libjansi.jnilib</pattern>
+						<pattern>org/eclipse/m2e/core/internal/lifecyclemapping/model/*.java</pattern>
```

Comment:
I wonder, why these files are detected as different? They are generated at the build server, nevertheless they should be equal since the build environment did not change since this morning?

Reply:
Maybe there is some timestamp or other noise added?
Note that the build server is a kubernetes cluster, so every build is a different (virtual) host.

Reply:
> Maybe there is some timestamp or other noise added? 

Indeed it is a timestamp generated. Thank you for that hint! I could have come up with that myself:

```
/*
 =================== DO NOT EDIT THIS FILE ====================
 Generated by Modello 1.5 on 2022-09-13 10:09:52,
 any modifications will be overwritten.
 ==============================================================
 */
```

> Note that the build server is a kubernetes cluster, so every build is a different (virtual) host.

I first assumed that this could be a reason, but for all other source files this wasn't an issue. And as long as all of them don't use a Windows OS line-endings should always be the same, just like encoding.